libcodechecker/libhandlers/store.py

```diff
@@ -246,6 +246,8 @@ def collect_file_hashes_from_plist(plist_file):
                                 plist_file, missing_files)
             elif f == 'metadata.json':
                 plist_report_files.append(os.path.join(input_path, f))
+            elif f == 'skip_file':
```

Comment:
You do the same thing here as at the 'metadata,json'. You can use the following form:
```py
elif f == 'metadata.json' or f == 'skip_file':
    plist_report_files.append(os.path.join(input_path, f))
```

Reply:
I think it is more visible what is packed into the zip file if we keep the separate `elif` branch.